datasets/us/ky/med_exclusions/us_ky_med_exclusions.yaml

```
@@ -0,0 +1,69 @@
+title: US Kentucky State Medicaid Terminations
+entry_point: crawler.py
+prefix: us-medky
+coverage:
+  frequency: daily
+  start: "2024-11-08"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Medical providers terminated from Kentucky State's Medicaid program.
+description: |
+  This dataset contains individuals and entites that have been terminated
+  from Kentucky's Medicaid Program. The following text explains the 
+  conditions and implications of such terminations or exclusions,
+  including the restrictions on reimbursement, re-enrollment criteria,
+  and responsibilities for Medicaid providers in screening employees and
+  contractors:
+
+   > An individual or entity terminated or excluded from participating in
+   > the Kentucky Medicaid program will not be reimbursed for services
+   > provided in any capacity or in any category under the Kentucky Medicaid
+   > program. The termination or exclusion remains in effect until the
+   > provider is removed from this list. Where applicable and when determined,
```

Could you add line breaks like these matching their paragraph breaks? It helps with the readability here and on the website. One blank line implies a paragraph break in markdown.

```suggestion
   > program. 
   >
   > The termination or exclusion remains in effect until the
   > provider is removed from this list. Where applicable and when determined,
```

---

I meant add the other paragraph breaks too :) I'll do it though but try and match their paragraphs next time.

---

Oh, sorry! I saw the suggestion and though you did for all of them

---

no stress :)